src/actions.js

```diff
@@ -67,3 +74,20 @@ export function loadRestaurants() {
     dispatch(setRestaurants(restaurants));
   };
 }
+
+export function loadDetailRestaurant(restaurantId) {
+  return async (dispatch, getState) => {
+    const {
+      selectedRestaurant: restaurant,
+    } = getState();
+
+    if (!restaurant) {
+      return;
+    }
+
+    const restaurantMenus = await fetchDetailRestaurant({
+      restaurantId: restaurantId,
+    });
```

Comment:
```suggestion
    const restaurantMenus = await fetchDetailRestaurant({
      restaurantId,
    });
```
이렇게 생략할 수 있을 것 같아요 ㅎㅎ